azerothcore/azerothcore-wotlk
# fix(Core/Spell): improve check in SPELL_EFFECT_CREATE_ITEM(_2)

fix from https://github.com/TrinityCore/TrinityCore/commit/18b36734f6ac235dbb7a37ba46d1c8d09d8171ab

## Changes Proposed:
-  Port TC handling of 
```
            case SPELL_EFFECT_CREATE_ITEM:
            case SPELL_EFFECT_CREATE_ITEM_2:
```


## Issues Addressed:
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/4295



## Tests Performed:
- Builds
- Tested in-game



## How to Test the Changes:

1. .add 37702 11
2. Right click the item to transform 10 into one big
3. fill your inventory with items so it is full
4. Click on Eternal Fire
5. Notice that it will now say "Inventory is full"

## Known Issues and TODO List:
- [x] This PR is dependant on that https://github.com/azerothcore/azerothcore-wotlk/pull/4297 is merged

## Target Branch(es):








 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
